.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+**Bug Report**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+How are you building/running PDC?
+
+```bash
+...
+```
+
+What is the running configuration file you use?
+
+```bash
+...
+```
+
+**Expected Behavior**
+
+A clear and concise description of what you expected to happen.
+
+**Software Environment**
+
+- version of PDC: [e.g. 0.3]
```

Comment:
ask for branch and commit hash is another option

Reply:
Done!